build.sbt

```diff
@@ -491,7 +514,6 @@ lazy val httpClientBackend = (projectMatrix in file("httpclient-backend"))
     }
   )
   .jvmPlatform(scalaVersions = List(scala2_13, scala3))
-  .settings(onlyJava11)
```

Comment:
Won't it break build for users of JDK 8? I think they'll have red builds.

Reply:
Yes, we require java11 for builds now. One of the modules required it anyway.